.github/workflows/ci/cmd/bot.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/gravitational/teleport/.github/workflows/ci"
```

Comment:
That makes sense. I figured since this code is only used with Github Actions it made sense to put it in `.github/workflows` where the GH actions workflow files reside.